codebuild_specs/createapi_canary_workflow.yml

```diff
@@ -0,0 +1,267 @@
+version: 0.2
+env:
+  shell: bash
+  compute-type: BUILD_GENERAL1_MEDIUM
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: build_linux
+      buildspec: codebuild_specs/build_linux.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+    - identifier: build_windows
```

Comment:
Nit: The test dependencies involved are only `build_linux` and `publish_to_local_registry`. The other tests such as `verify_yarn_lock`, `mock_e2e_tests`, can be removed if the test mainly focuses on the region e2e tests, since those are already included in the PR workflow